doc/news/changes.h

```diff
@@ -89,6 +95,11 @@ inconvenience this causes.
 
 
 <ol>
+  <li> New: Support for Abaqus mesh files has been added to the Grid_In class.
```

Comment:
Yes, sorry I thought I'd corrected all of those typos. Thanks for the doxygen tip!